packages/paste-website/src/component-examples/ComboboxPrimitiveExamples.ts

```diff
@@ -104,9 +103,9 @@ render(
 
 export const multiSelectExample = `const items = ['Alert', 'Anchor', 'Button', 'Card', 'Heading', 'List', 'Modal', 'Paragraph'];
 
-const BasicMultiCombobox = () => {
+const MultipleSelectCombobox = () => {
   const seed = useUIDSeed();
-  const [filteredItems, setFilteredItems] = React.useState([...items]);
+  const [filteredItems, setFilteredItems] = React.useState([...items].sort());
```

Comment:
```suggestion
  const [filteredItems, setFilteredItems] = React.useState(items);
```

Can add the sort to the tail end of line 104 and not need to do anything fancy here

Reply:
❓ Is there a technical concern with line 108?

FWIW I spread the items array into the state hook because I've seen bugs from not cloning objects (in this case it's just a shallow clone bc it's just an array of strings).

I just quickly [spun up this sandbox that demonstrates this](https://codesandbox.io/s/original-state-when-passed-an-array-zsps7?file=/src/App.tsx).

if you press the buttons with the sandbox terminal open, you can see that when the original array is passed, it's super easy to accidentally modify the original object. Worth noting, that this could be solved in the specific instance of updating the state by just spreading the old state in with the new.